css-pseudo-classes/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h1>Hover Button</h1>
+    <button>I'm a button!</button>
+    <h1>Striped Table</h1>
+    <table>
+      <thead>
+        <tr id="header">
+          <th>Student Name</th>
+          <th>Course</th>
+          <th>Grade</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td style="font-weight: normal">Ada Lovelace</td>
+          <td style="font-weight: normal">Web Development</td>
+          <td style="font-weight: normal">100</td>
```

Comment:
I am pretty sure `td` elements have a `font-weight: 400` by default, which is already kind of the normal